### PR TITLE
#1621 : fixed list formatting

### PR DESCRIPTION
**Pull request by Swikar**
I found listing problem in three files.

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/website/1645)

### CODE REVIEW

**Comment by sarahmaddox** [comment]
/ok-to-test
/assign

@Swikar Is this PR ready for review? I'm asking because it has **WIP** in the PR description, which usually means the PR author is still busy and not yet ready for review.

File: content/docs/gke/deploy/delete-cli.md [hunk 1]
```diff
@@ -40,15 +40,15 @@ For further background about the above settings, see the guide to
 Run the following commands to delete your deployment and reclaim all GCP
 resources:
 
-```
-# If you want to delete all the resources, including storage:
-kfctl delete -f ${CONFIG_FILE} --delete_storage
+  ```
```
Comment (sarahmaddox) [inline content/docs/gke/deploy/delete-cli.md]: I don't think this indentation change is necessary. The page looks good on master (https://master.kubeflow.org/docs/gke/deploy/delete-cli/) but in the preview for this PR, there are now unnecessary empty spaces in the code snippet: https://deploy-preview-1645--competent-brattain-de2d6d.netlify.com/docs/gke/deploy/delete-cli/

File: content/docs/gke/deploy/deploy-cli.md [hunk 2]
```diff
@@ -16,8 +16,8 @@ Before installing Kubeflow on the command line:
 
 1. Ensure you have installed the following tools:
   
-  * [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/).
-  * [gcloud](https://cloud.google.com/sdk/). If you already have `gcloud`
+    * [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/).
```
Comment (sarahmaddox) [inline content/docs/gke/deploy/deploy-cli.md]: This change also doesn't seem necessary, as the page is working OK on master.

File: content/docs/gke/deploy/deploy-cli.md [hunk 3]
```diff
@@ -195,11 +195,11 @@ using [yq](https://github.com/mikefarah/yq/releases):
 
 1. Run the `kfctl build` command to generate kustomize and GCP Deployment manager configuration files for your deployment:
 
-  ```
-  mkdir -p ${KF_DIR}
-  cd ${KF_DIR}
-  kfctl build -V -f ${CONFIG_URI}
-  ```
+    ```
```
Comment (sarahmaddox) [inline content/docs/gke/deploy/deploy-cli.md]: Thanks, this is perfect!

File: content/docs/gke/deploy/deploy-cli.md [hunk 4]
```diff
@@ -215,9 +215,9 @@ using [yq](https://github.com/mikefarah/yq/releases):
 
 1. Run the `kfctl apply` command to deploy Kubeflow:
 
-  ```
-  kfctl apply -V -f ${CONFIG_FILE}
-  ```
+    ```
```
Comment (sarahmaddox) [inline content/docs/gke/deploy/deploy-cli.md]: This is good too.

File: content/docs/gke/deploy/deploy-cli.md [hunk 5]
```diff
@@ -253,23 +253,23 @@ Follow these steps to access the Kubeflow central dashboard:
 1. Enter the following URI into your browser address bar. It can take 20
   minutes for the URI to become available:
 
-  ```
-  https://<KF_NAME>.endpoints.<project-id>.cloud.goog/
-  ```
+    ```
```
Comment (sarahmaddox) [inline content/docs/gke/deploy/deploy-cli.md]: This is good, and so is the rest of this file.

**Comment by sarahmaddox** [comment]
Preview of one of the pages: https://deploy-preview-1645--competent-brattain-de2d6d.netlify.com/docs/gke/deploy/delete-cli/

**Comment by Swikar** [comment]
@sarahmaddox it is ready for review now

File: content/docs/gke/deploy/project-setup.md [hunk 6]
```diff
@@ -25,13 +25,13 @@ Follow these steps to set up your GCP project:
...
   specified APIs are enabled:
 
-  * [Compute Engine API](https://console.cloud.google.com/apis/library/compute.googleapis.com)
-  * [Kubernetes Engine API](https://console.cloud.google.com/apis/library/container.googleapis.com)
-  * [Identity and Access Management (IAM) API](https://console.cloud.google.com/apis/library/iam.googleapis.com)
-  * [Deployment Manager API](https://console.cloud.google.com/apis/library/deploymentmanager.googleapis.com)
-  * [Cloud Resource Manager API](https://console.developers.google.com/apis/library/cloudresourcemanager.googleapis.com)
-  * [Cloud Filestore API](https://console.developers.google.com/apis/library/file.googleapis.com)
-  * [AI Platform Training & Prediction API](https://console.developers.google.com/apis/library/ml.googleapis.com)
+    * [Compute Engine API](https://console.cloud.google.com/apis/library/compute.googleapis.com)
```
Comment (sarahmaddox) [inline content/docs/gke/deploy/project-setup.md]: Could you talk me through the reason for updating this file? The page on master seems good without the update: https://master.kubeflow.org/docs/gke/deploy/project-setup/

**Comment by sarahmaddox** [comment]
Fixes https://github.com/kubeflow/website/issues/1621

**Comment by sarahmaddox** [comment]
Thanks for all the hard work on this @Swikar. It looks good, and the broken lists are fixed.

/lgtm
/approve

**Pull request merged**